.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.ts text eol=lf
+*.js test eol=lf
```

Comment:
Windows test fails regarding to prettier on the end of line break style. See [commit](https://github.com/googleapis/gcs-resumable-upload/pull/329/commits/fe6d378d4a10bdf929e3bbb75174211aef3a52db)
Added .gitattributes to normalize line endings to LF on checkin, and do NOT convert to CRLF when checking-out on Windows.

Not sure the best practice here. Open for any suggestion. Thanks :) 🙏 😊 
@bcoe @JustinBeckwith @alexander-fenster